src/auro-toast.js

```diff
@@ -64,12 +71,26 @@ export class AuroToast extends LitElement {
     return [styleCss];
   }
 
+  /**
+   * @private
+   * @param {string} svgContent - The imported svg icon.
+   * @returns {string} - The html template for the icon.
+   */
+  generateIconHtml(svgContent) {
+    const dom = new DOMParser().parseFromString(svgContent, 'text/html'),
+      svg = dom.body.firstChild;
+
+    return this.noIcon
+      ? html``
+      : html`<div class="icon">${svg}</div>`;
+  }
+
   /**
    * @private
    * @returns {void}
    */
   handleOnClose() {
-    this.visible = false;
+    this.fadeOutToast();
```

Comment:
Would we want to call `closeToast` here as well? I would think we don't really care about fading out the toast when a user clicks the X, we just want it to disappear.